editors/code/src/util.ts

```diff
@@ -1,7 +1,9 @@
-import * as lc from "vscode-languageclient";
+import * as lc from "vscode-languageclient/node";
 import * as vscode from "vscode";
 import { strict as nativeAssert } from "assert";
 
+//import * as prot from "vscode-languageclient/lib/node'
```

Comment:
Same here, etc..

Reply:
The functionality that was in vscode-languageclient is now in vscode-languageclient/node in the latest version.